src/events/EventsConfig.js

```diff
@@ -164,15 +164,17 @@ const eventsConfig = {
       name: "events0", 
       popupTemplate: popupTemplate,
       showLegend: true, 
-      outFields: ['eventType', 'eventTimestamp', 'eventExpirationTimestamp']
+      outFields: ['eventType', 'eventTimestamp', 'eventExpirationTimestamp'],
+      refreshInterval: 10
     },// baselineWhereCondition:eventsBaselineWhereCondition},
     {
       id: 1, 
       type: "live", 
       name: "events1",
       popupTemplate: popupTemplate,
       showLegend: false,
-      outFields: ['eventType', 'eventTimestamp', 'eventExpirationTimestamp']
+      outFields: ['eventType', 'eventTimestamp', 'eventExpirationTimestamp'],
+      refreshInterval: 10
     },// baselineWhereCondition:eventsBaselineWhereCondition},
```

Comment:
So now you can add `refreshInterval` in minutes if you want to enable auto-refresh. By default it's 0. 
I've deleted the whole 'live' stuff :)